docs/domains/blas/blas.rst

```diff
@@ -13,5 +13,3 @@ oneMKL provides DPC++ interfaces to the Basic Linear Algebra Subprograms (BLAS)
     blas-level-3-routines.rst
     blas-like-extensions.rst
 
-
-**Parent topic:** :ref:`onemkl_dense_linear_algebra`
```

Comment:
Why is this removed?

Reply:
Hi @mmeterel , it was removed because it was a duplicate file and not found in any toctree. I discussed with @mkrainiuk offline prior to the PR and she said it was OK to remove. The proper file is located at docs/domains/dense_linear_algebra.rst.

Docs build without errors now. Let me know if you have other concerns.

Regards,
John

Reply:
Hi @JOCwriter what i meant is, why the "parent topic" is removed.

Reply:
Hi @mmeterel , thank you for clarifying. There is an initiative underway to align all library components to have the same look/feel and the "Parent Topic" entry is not standardized. If you look at oneDNN opensource docs for example (https://oneapi-src.github.io/oneDNN/v2/index.html), this is the direction we are moving for oneMKL. I will be making future ongoing updates to apply standard templates to all of the topics in this doc set, but this just got picked up in this PR because I was doing other work in the topic.

Reply:
@JOCwriter thanks for clarification.